src/palakSJun20/DifferIndexFromArray.java

```diff
@@ -0,0 +1,75 @@
+/*Program 2 : Find first differ index from given arrays. 
+                   i/p :- arr1: {10,2,9,14,3}
+                          arr2: {10,2,18,14,3}
+                   o/p:- Values are not matching at index -> 2 
+                   
+Program 3: Find all differ index from given arrays. 
+                 i/p:- arr1: {10,2,9,14,3}
+                        arr2: {10,2,18,14,13}
+                o/p:- Values are not matching at index -> 2
+                         Values are not matching at index -> 4                  
+Program 4: Find max value at differed index.
+              i/p:- arr1: {10,2,9,14,3}
+                     arr2: {10,2,18,14,3}
+              o/p :- Values are not matching at index -> 2, From (9,18) max value is 18
+                                        */
+
+package palakSJun20;
+
+public class DifferIndexFromArray {
+	
+	//Find first differ index from given arrays. 
+	void findFirstDifferIndex(int[] array1 , int[] array2){
+		
+		if (array1.length == array2.length) {
+			for (int i = 0; i < array1.length; i++) {
+					if(array1[i] != array2[i]) {
+						System.out.println("Values are not matching at index " + i);
+						
+				}
+			}
+		}
+	}
+	
+	//Find all differ index from given arrays. 
+	void findAllDifferIndex(int[] array3 , int[] array4){
+		  findFirstDifferIndex(array3, array4);
+	}
+	
+	//Find max value at differed index.
+	void findMaxValueAtDifferIndex(int[] array5, int[] array6){
+		if (array5.length == array6.length) {
+			for (int i = 0; i < array5.length; i++) {
+					if(array5[i] != array6[i]) {
+						System.out.println("Values are not matching at index " + i);
+						if(array5[i] > array6[i])
+							System.out.println("From "+ "(" + array5[i] +","+ array6[i]  + ")" + " Max Value is" + array5[i]);
+						else 
```

Comment:
The printing statement is not as expected.
System.out.println("Values are not matching at index -> " + index + ", From (" + arr1[index] + ","
							+ arr2[index] + ") max value is " + max);
You should write this line after if block.

if (arr1.length == arr2.length) {
			for (int index = 0; index < arr1.length; index++) {
				int max = arr2[index];
				if (arr1[index] != arr2[index]) {
					if (arr1[index] > arr2[index])
						max = arr1[index];
					System.out.println("Values are not matching at index -> " + index + ", From (" + arr1[index] + ","
							+ arr2[index] + ") max value is " + max);
				}
			}
		} else
			System.out.println("Both array's length is different.");

Reply:
as per suggested changes has been updated.